src/lib/filesystem.ts

```diff
@@ -1,28 +1,13 @@
-// Load the files selected with an input type="file" into the origin
-// private filesystem in order to make them easy to retrieve in the future
-export async function copyToPrivateFileSystem(target: HTMLInputElement) {
-	const files = target.files;
-
-	if (!files) {
-		return [];
-	}
-
+export async function copyToPrivateFileSystem(name: string, file: File) {
```

Comment:
Changed to handle a single file and to get the name as input.

This way we can use the `id` as file name, making possible to store multiple files with the same name.